.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "webcam-capture-driver-native"]
```

Comment:
we should probably add a step to the readme to `mvn install` in the `webcam-capture-driver-native` directory